### PR TITLE
Volume rendering performance

### PR DESCRIPTION
**Pull request by thewtex**
CC: @martinken @jourdain

### CODE REVIEW

**Comment by jourdain** [comment]
Is that much faster? 
BTW, you need to reformat your code `npm run reformat`

**Comment by thewtex** [comment]
Yes, it results in a overall volume rendering performance improvement of >15%. The array min/max operations take ~1/3 of what they did before.

Fixing the formatting. Thanks for the `npm run reformat` tip!

File: Sources/Common/Core/Math/index.js [hunk 1]
```diff
@@ -41,6 +42,33 @@ const Pi = () => Math.PI;
...
+
+function arrayMin(arr) {
+  let minValue = Infinity;
+  for (let i = 0, len = arr.length; i < len; i += MAX_FUNCTION_ARGUMENTS) {
+    const submin = Math.min.apply(
+      null,
+      arr.slice(i, Math.min(i + MAX_FUNCTION_ARGUMENTS, len))
+    );
+    minValue = Math.min(submin, minValue);
+  }
```
[the remark this answers is not in the document]
Comment (martinken) [reply Sources/Common/Core/Math/index.js]: I always forget to submit my reviews

[the remark this answers is not in the document]
Comment (thewtex) [reply Sources/Common/Core/Math/index.js]: Good suggestion, @martinken . Yes, this has even better performance. I will follow-up in an another PR.

[the remark this answers is not in the document]
Comment (jourdain) [reply Sources/Common/Core/Math/index.js]: can you do another test?

```
let count = arr.length;
while (count--) {
   minValue = minValue < arr[count] ? minValue : arr[count];
}
```

[the remark this answers is not in the document]
Comment (jourdain) [reply Sources/Common/Core/Math/index.js]: you can do also a `console.time('min') / console.endTime('min')` to see if you find any delta T.

**Comment by thewtex** [comment]
Timings:

Existing Math.min: ~210ms for `arrayMin`, ~2712ms for entire volume rendering initialization 
Comparison operator: ~35ms for `arrayMin`, ~2070ms for entire volume rendering initialization 
Ternary operator: ~37ms for `arrayMin`, ~2125ms for entire volume rendering initialization

**Pull request merged**